common/scripts/shell-wrapper.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# The "default parameters to entrypoint" and "exec" forms of Docker's CMD
+# instruction don't run in a shell and therefore variables (e.g. $HOME) are not
+# evaluated. This script evaluates the CMD values in a shell and then replaces
+# the shell process with the evaluated command.
+eval exec $@
```

Comment:
We should us `"$@"` instead of `$@` here so that quoted parameters containing spaces are kept together.

In the `$@` case, `shell-wrapper.sh a "b1 b2" c` results in `eval exec a b1 b2 c`. If we use `"$@"`, it results in `eval exec ` "b1 b2" c` instead, which is what we want.